### ISSUE TITLE
Saving or loading any .leo workbook with todo.py icons on nodes causes several 'cannot load image: None' messages printed to log

### ISSUE THREAD

**Issue by gatesphere** [leo-editor/leo-editor#3284]
My environment:

Leo 6.7.3-devel, devel branch, build e2c9fa06e0
2023-04-17 20:36:03 -0500
Python 3.10.6, PyQt version 6.3.1
Windows 10 AMD64 (build 10.0.19044) SP0

(Note: I did not experience this issue in my Arch Linux environment, with same theme settings)

Steps to reproduce:

  - load or save any .leo workbook containing todo icons on nodes

Bug symptoms:

  - Many messages in the log pane -- nothing printed to the console:

```
can not load image: None
can not load image: None
can not load image: None
can not load image: None
can not load image: None
can not load image: None
can not load image: None
can not load image: None
can not load image: None
<...>
```

After these messages flood the log pane, the UI hangs for quite a few seconds before allowing interaction again.

**Comment by gatesphere** [comment]
A bit of poking indicates Leo is having trouble finding the todo icons:

<redacted>/leo-editor/leo/core/cleo/chkblk.png

I have no idea why Leo is now looking in core instead of Icons.

**Comment by tbpassin** [comment]
`g.os_path_join()` is now prepending the outline's location, which is not what it should do:

```
path = g.os_path_join('a', 'b')
g.es(path)

# C:/Users/tom/.leo/a/b
```

**Comment by edreamleo** [comment]
@tbpassin Thanks for this. I'll restore the legacy `g.os_path_join`.

**Comment by gatesphere** [comment]
Can confirm that devel commit cae2ee8728 fixes this issue in my Windows
environment.  Thanks for the quick fix.

**Comment by edreamleo** [comment]
On Tue, Apr 18, 2023 at 10:25 AM Jacob Peck ***@***.***>
wrote:

> Can confirm that devel commit cae2ee8728 fixes this issue in my Windows
> environment. Thanks for the quick fix.
>

You're welcome. Actually, I pushed that commit by mistake. It's been one of
those days...

I've just merged PR #3286 into devel, and that should retain the fix of
your problem.

Please retest devel.

Edward

**Comment by gatesphere** [comment]
Confirmed -- still fixed in commit ca92d1f510.

Thanks.

**Comment by edreamleo** [comment]
@gatesphere You're welcome. Thanks for the continued testing.
